flytekit/clis/sdk_in_container/register.py

```diff
@@ -0,0 +1,164 @@
+)
+@click.option(
+    "-i",
+    "--image",
+    "image_config",
+    required=False,
+    multiple=True,
+    type=click.UNPROCESSED,
+    callback=ImageConfig.validate_image,
+    default=[DefaultImages.default_image()],
+    help="A fully qualified tag for an docker image, e.g. somedocker.com/myimage:someversion123. This is a "
+    "multi-option and can be of the form --image xyz.io/docker:latest "
+    "--image my_image=xyz.io/docker2:latest. Note, the `name=image_uri`. The name is optional, if not "
+    "provided the image will be used as the default image. All the names have to be unique, and thus "
+    "there can only be one --image option with no name.",
+)
+@click.option(
+    "-o",
+    "--output",
+    required=False,
+    type=click.Path(dir_okay=True, file_okay=False, writable=True, resolve_path=True),
+    default=".",
+    help="Directory to write the output zip file containing the protobuf definitions",
+)
+@click.option(
+    "-d",
+    "--destination-dir",
+    required=False,
+    type=str,
+    default="/root",
+    help="Directory inside the image where the tar file containing the code will be copied to",
+)
+@click.option(
+    "--service-account",
+    required=False,
+    type=str,
+    default="",
+    help="Service account used when creating launch plans",
+)
+@click.option(
+    "--raw-data-prefix",
+    required=False,
+    type=str,
+    default="",
+    help="Raw output data prefix when creating launch plans, where offloaded data will be stored",
+)
+@click.option(
+    "-v",
+    "--version",
+    required=False,
+    type=str,
+    help="Service account used when creating launch plans",
+)
+@click.argument("package-or-module", type=click.Path(exists=True, readable=True, resolve_path=True), nargs=-1)
+@click.pass_context
+def register(
+    ctx: click.Context,
+    project: str,
+    domain: str,
+    image_config: ImageConfig,
+    output: str,
+    destination_dir: str,
+    service_account: str,
+    raw_data_prefix: str,
+    version: typing.Optional[str],
+    package_or_module: typing.Tuple[str],
+):
+    """
+    This command is similar to package but instead of producing a zip file, all your Flyte entities are compiled,
+    and then sent to the backend specified by your config file. Think of this as combining the pyflyte package
+    and the flytectl register step in one command. This is why you see switches you'd normally use with flytectl
+    like service account here.
+
+    Note that this command runs "fast" register by default. Future work to come to add a non-fast version.
+    This means that a zip is created from the detected root of the packages given, and uploaded. Just like with
+    pyflyte run, tasks registered from this command will download and unzip that code package before running.
+    """
+
+    pkgs = ctx.obj[constants.CTX_PACKAGES]
+    if not pkgs:
+        cli_logger.debug("No pkgs")
+    if pkgs:
+        raise ValueError("to do, please implement")
```

Comment:
how would that look like?

Reply:
probably use in lieu of the args provided?  maybe... let me change the text.  maybe we just leave it unimplemented.